index.js

```diff
@@ -1,22 +1,39 @@
 require('dotenv').load();
+var qs = require('qs');
 
 var stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
 
-exports.handler = function(event, context) {
+exports.handler = function(event, context, callback) {
+  console.log("event: \n" + JSON.stringify(event, null, 4));
+  var data = qs.parse(event.body);
+  console.log("body: \n" + JSON.stringify(data, null, 4));
 
-  stripe.charges.create({
-    amount:        event.amount,
-    source:        event.source,
-    currency:      event.currency || 'usd',
-    description:   event.description || 'Stripe payment '+event.order_id,
-    receipt_email: event.receipt_email || null
+  var customer = stripe.customers.create({
+    source: data.stripeToken,
+    email: data.stripeEmail,
+    description: "Customer for " + data.stripeEmail,
+    plan: 'littleFriendIG500'
   }, function(err, charge) {
-    if (err && err.type === 'card_error') {
-      context.fail(new Error(err.message));
-    } else if(err){
-      context.fail(err);
+    if (err) {
+      console.log(err.type + ": " + err.message + ". (code: " + err.code + ")");
+      context.succeed({
+        "statusCode": 302,
+        "headers": { "Location" : "http://littlefriend.co/error?type=" + err.type },
+        "body": ""
+      });
     } else {
-      context.succeed({ status: charge.status, success : true });
+      context.succeed({
+        "statusCode": 302,
+        "headers": { "Location" : "http://littlefriend.co/success" },
+        "body": ""
+      });
     }
+    //if (err && err.type === 'card_error') {
+    //  context.fail(new Error(err.message));
+    //} else if(err){
+    //  context.fail(err);
+    //} else {
+    //  context.succeed({ status: charge.status, success : true });
+    //}
```

Comment:
I wouldn't be able to pull these changes in with this PR; they appear to be specific to your use case.

Reply:
Oh, ugh. I keep forgetting that PRs aren't frozen in time, so you can't continue to push to your own branch. Can you pull in only that first commit? Or do I need to re-fork?

Reply:
You would need to make a new PR with just the one commit that adds the error logging.